src/lib/wind_estimator/python/wind_est_derivation.py

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Nov  1 19:14:39 2016
+
+@author: roman
+"""
+
+from sympy import *
+
```

Comment:
Glad to see the use of sympy. Any chance you can include an ipython notebook with the rendered equations? I think it would be worthwhile since github renders them.